src/Parser.js

```diff
@@ -24,6 +24,9 @@
 class Parser {
     /**
      * Construct a new plugin.
+     *
+     * @param {Object|undefined} options options for this instance of the
```

Comment:
'options' repetition?

Reply:
The first options names the parameter to the method, and the second one is part of the description sentence